### ISSUE TITLE
iOS / xcode10.1

### ISSUE THREAD

**Issue by rvleyden** [symeonmattes/pjsip#5]
Hello,

where can I set the following in Xcode 10.1 ? : "After installation you need to add manually in Build settings the following configurations a. Apple LLVM 8.1 Preprocessing->Preprocessor Macros->(both debug/release) PJ_AUTOCONF=1 b. Search paths->Header Search Paths -> (both debug/release in cordova plugin) i. fulllocation/pjsua/pjmedia ii. fulllocation/pjsua/pjnath iii. fulllocation/pjsua/pjlib-util iv. fulllocation/pjsua/pjlib v. fulllocation/pjsua/pjsip"

**Comment by symeonmattes** [comment]
Hi,

It's a tab called build settings. Check the image:
https://cdn-images-1.medium.com/max/1200/1*JH3lJ0krxRiZepKpnzwS0g.png

**Comment by rvleyden** [comment]
Thanks, but I still do not onderstaand what Value's to put in and on which parameters. I'm sorry but I am not so well know with xcode and project parameters.

**Comment by symeonmattes** [comment]
Check the images:
https://raw.githubusercontent.com/BranchMetrics/react-native-branch-deep-linking/master/docs/assets/8-add-preprocessor-macro.png
https://blogs.wcode.org/wp-content/uploads/2014/11/Screen-Shot-2014-11-19-at-09.30.10-2.png

In Xcode the build settings tab is supposed to be setting for the compiler. Normally you will find sections like Apple LLVM 8.1 etc, Search paths etc. There you should add (with the + sign) the parameters I mention.

**Comment by rvleyden** [comment]
Is the setting correct as marked in my screenshot ?
![image](https://user-images.githubusercontent.com/24704792/53873115-f3bd7180-3fff-11e9-952d-8a640b9c63df.png)

**Comment by symeonmattes** [comment]
It seems to be ok, but the value is not 1 b, it's 1. In the manual it's like that:
After installation you need to add manually in Build settings the following configurations 
a. Apple LLVM 8.1 Preprocessing->Preprocessor Macros->(both debug/release) PJ_AUTOCONF=1 
b. Search paths->Header Search Paths -> (both debug/release in cordova plugin)
 i. fulllocation/pjsua/pjmedia 
ii. fulllocation/pjsua/pjnath 
iii. fulllocation/pjsua/pjlib-util 
iv. fulllocation/pjsua/pjlib v. fulllocation/pjsua/pjsip

**Comment by rvleyden** [comment]
Is it possible to send me a sample project (ionic) where the settings are working for Ios ?

Thanks

**Comment by rvleyden** [comment]
for the following : 

i. fulllocation/pjsua/pjmedia
ii. fulllocation/pjsua/pjnath
iii. fulllocation/pjsua/pjlib-util
iv. fulllocation/pjsua/pjlib v. fulllocation/pjsua/pjsip

Do you mean with fulllocation the path to the pjsua directory in de

**Comment by symeonmattes** [comment]
yes...it's the full path. I hadn't found another way to deal with this cumbersome.
